sql/main.sql

```diff
@@ -1,9 +1,23 @@
 -- Default PostgREST OpenAPI Specification
 
+-- This is the one we should be calling
+create or replace function callable_root() returns jsonb as $$
+	-- Calling this every time is inefficient, but it's the best we can do until PostgREST calls it when it updates the server config
+	CALL set_server_from_configuration();
+	SELECT get_postgrest_openapi_spec(
+		schemas := string_to_array(current_setting('pgrst.db_schemas', TRUE), ','),
+		postgrest_version := current_setting('pgrst.version', TRUE),
+		proa_version := '0.1'::text, -- FIX: needs to be updated; put into config, and have Makefile update
```

Comment:
I think these `FIX`es should be preceded by `TODO`, most IDEs highlight/summarize the TODOs nicely. Something like:

```suggestion
		proa_version := '0.1'::text, -- TODO: (FIX) needs to be updated; put into config, and have Makefile update
```

Reply:
Did s/FIX/TODO/ and s/FIXMVP/TODO: (MVP)/